tests/spec/fs.write.spec.js

```diff
@@ -1,24 +1,25 @@
-var util = require('../lib/test-utils.js');
-var expect = require('chai').expect;
+'use strict';
+const util = require('../lib/test-utils.js');
+const expect = require('chai').expect;
 
 describe('fs.write', function() {
   beforeEach(util.setup);
   afterEach(util.cleanup);
 
   it('should be a function', function() {
-    var fs = util.fs();
+    let fs = util.fs();
```

Comment:
fs can be const as it isnt changed in this scope
```suggestion
    const fs = util.fs();
```